build.gradle.kts

```diff
@@ -68,10 +68,11 @@ dependencies {
   kover(projects.arrowContinuations)
   kover(projects.arrowCore)
   kover(projects.arrowCoreRetrofit)
+  // kover(projects.arrowCoreSerialization)
   kover(projects.arrowFxCoroutines)
   kover(projects.arrowFxStm)
   kover(projects.arrowOptics)
-  kover(projects.arrowOpticsKspPlugin)
+  // kover(projects.arrowOpticsKspPlugin)
```

Comment:
I think this can be removed instead of comment?
```suggestion
```

Reply:
I’d rather keep it alongside the comment in the corresponding section